discovery/db_discovery.go

```diff
@@ -212,3 +221,21 @@ func pmTicketParams(params *net.TicketParams) *pm.TicketParams {
 		Seed:              new(big.Int).SetBytes(params.Seed),
 	}
 }
+
+func (dbo *DBOrchestratorPoolCache) PollOrchestratorInfo(bcast common.Broadcaster) {
+	ticker := getTicker()
+	go func() {
+		for range ticker.C {
+			orchs, err := dbo.store.SelectOrchs(
+				&common.DBOrchFilter{
+					CurrentRound: dbo.rm.LastInitializedRound(),
+				},
+			)
+
+			if err != nil {
+				glog.Errorf("could not get orchestrators from database: %v", err)
+			}
+			dbo.cacheDBOrchs(bcast, orchs)
```

Comment:
Need to handle error here

Reply:
I just checked and we're never actually returning an error from `cacheDBOrchs` so instead I opt for removing `error` as return param